discovery-infra/tests/base_test.py

```diff
@@ -258,3 +285,56 @@ def wait_for_nodes_status_installing_or_installed(
                       consts.NodesStatus.INSTALLED],
             interval=30,
         )
+
+    def setup_hosts(self, cluster_id, api_client, node_controller):
+        self.generate_and_download_image(cluster_id=cluster_id,
+                                         api_client=api_client)
+        node_controller.start_all_nodes()
+        self.wait_until_hosts_are_registered(cluster_id=cluster_id,
+                                             api_client=api_client)
+        return api_client.get_cluster_hosts(cluster_id=cluster_id)
+    
+    def prepare_for_installation(self, cluster_id, api_client, node_controller):
+        self.set_network_params(cluster_id=cluster_id,
+                                api_client=api_client,
+                                controller=node_controller)
+    
+    def expect_ready_to_install(self, cluster_id, api_client):
+        self.wait_until_cluster_is_ready_for_install(cluster_id=cluster_id,
+                                                     api_client=api_client)
+    
+    def start_installation(self, cluster_id, api_client):
+        self.start_cluster_install(cluster_id=cluster_id,
+                                   api_client=api_client)
+        self.wait_until_cluster_starts_installation(cluster_id=cluster_id,
+                                                    api_client=api_client)
+        hosts = api_client.get_cluster_hosts(cluster_id=cluster_id)
+        return {h["id"]: h["role"] for h in hosts}
+
+    def assert_assignments(self, actual_assignments, expected_assignments):
```

Comment:
IMO helper functions shouldn't have asserts in order to support negative scenarios
Moreover, I don't understand the meaning of `assert_assignments`. Please have meaningful function names

Reply:
Done
In the long term I don't agree with you. It's not unit tests and it would become a nightmare to understand each and every assertion in the entire context of a more complex test. In the short-term of those initial tests you are right so I'm changing it